Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView+AdvancedView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TransactionDetailsView+AdvancedView.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension TransactionDetailsView {
+
+    struct AdvancedView: View {
+        let transaction: TransactionViewModel
+
+        static let height: CGFloat = 48
+
+        @ViewBuilder var body: some View {
```

Comment:
something new for me :)

Reply:
learned from WWDC20 - this is a proper way to handle multiple return types, because this creates a view builder tuple, instead of manually wrapping in the ZStack. @MouazAlzahabi please take a note, too 😄

Reply:
nice